po/hi_IN.po

```diff
@@ -0,0 +1,622 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR Andy Alt
+# This file is distributed under the same license as the rmw package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: rmw 0.7.04\n"
+"Report-Msgid-Bugs-To: andy400-dev@yahoo.com\n"
+"POT-Creation-Date: 2019-08-12 15:19-0500\n"
+"PO-Revision-Date: 2020-04-18 12:26+0530\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n==0 || n==1);\n"
+"X-Generator: Poedit 2.3\n"
+"Last-Translator: \n"
+"Language: hi_IN\n"
+
+#: src/config_rmw.c:70
+msgid ""
+"\n"
+"# A folder can use the $UID variable.\n"
+msgstr ""
+"\n"
+"# folder $UID variable ka istemal karta hai\n"
+
+#: src/config_rmw.c:81
+msgid ""
+"\n"
+"# How many days should files be allowed to stay in the waste folders "
+"before\n"
+"# they are permanently deleted\n"
+"#\n"
+"# use '0' to disable purging\n"
+"#\n"
+msgstr ""
+" \n"
+"# Kitne dino tak failonko ko pehle apshist folder may rehne ki anumati "
+"di jani chahiye?\n"
+"# Yeh sthayi roop se hata di jati hai\n"
+"\n"
+"# mitane ke liye '0' ka upyog karen\n"
+"#\n"
+
+#: src/config_rmw.c:56
+msgid ""
+"\n"
+"# If you would like this to be your primary trash folder (which usually "
+"means\n"
+"# that it will be the same as your Desktop Trash folder) be sure it "
+"precedes\n"
+"# any other WASTE folders listed in the config file\n"
+"#\n"
+"# If you want it checked for files that need purging, simply uncomment\n"
+"# The line below. Files you move with rmw will go to the folder above "
+"by\n"
+"# default.\n"
+"#\n"
+"# Note to OSX and Windows users: sending files to 'Desktop' trash\n"
+"# doesn't work yet\n"
+"#\n"
+msgstr ""
+"\n"
+"# yadi aap chahte hai ki yeh aapka prathamik kachra folder ho(jiska "
+"aamtor par matlab hota hai\n"
+"# ki yeh aapko desktop trash folder ke saamna hoga) sunischit karen ki "
+"yeh pehle\n"
+"# config file may suchibandh bhi anya apshist folder\n"
+"#\n"
+"# yadi aap chahte hai k yeh unn failon ke liye jaanch kiya jaye jinhe "
+"mitane ki abyasakta hai, toh bas tipanni na kare \n"
+"# niche ki rekha| failen jo aap rmw hatayega to woh upar wale folder ko "
+"jayega   \n"
+"# default।\n"
+"#\n"
+"# OSX aur Windows upyogi hai to dhyan de: 'Desktop' kachre keliye "
+"folder bhejna  \n"
+"# abhi tak kaam nahi karta hai\n"
+"#\n"
+
+#: src/config_rmw.c:73
+msgid ""
+"\n"
+"# Removable media: If a folder has ',removable' appended to it, rmw\n"
+"# will not try to create it; it must be initially created manually. If\n"
+"# the folder exists when rmw is run, it will be used; if not, it will "
+"be\n"
+"# skipped Once you create \"example_waste\", rmw will automatically "
+"create\n"
+"# example_waste/info and example_waste/files\n"
+msgstr ""
```

Comment:
Between L93 and L94 there needs to be a "\n" to match with what's in the "msgid" section above.